src/PostgREST/SchemaCache.hs

```diff
@@ -155,9 +158,6 @@ querySchemaCache AppConfig{..} = do
   reps    <- SQL.statement schemas $ dataRepresentations prepared
   mHdlers <- SQL.statement schemas $ mediaHandlers pgVer prepared
   tzones  <- SQL.statement mempty $ timezones prepared
-  _       <-
-    let sleepCall = SQL.Statement "select pg_sleep($1)" (param HE.int4) HD.noResult prepared in
-    whenJust configInternalSCSleep (`SQL.statement` sleepCall) -- only used for testing
```

Comment:
I remember I put this explicitly at the end to add a test for https://github.com/PostgREST/postgrest/pull/2810. Is it necessary to change it to the beginning?

I wonder if this test https://github.com/PostgREST/postgrest/blob/main/test/io/test_io.py#L1024-L1051 continues to work as intended.

If it doesn't, maybe I can add another sleep here.

---

I think there's a nice way to maintain these sleeps, in case we also need in other place of the code. We can keep using this "observation" pattern I've been doing for logs and metrics. We would just need an `observer $ Observation StartQuerySCache/EndQuerySCache` and do the sleeps in another module.

Reply:
I will make a note to do this change in another PR.